### PR TITLE
Toggle k-th bit of a number from Right in C++

### PR DESCRIPTION
**Pull request by mansi-9**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
 
Yes

### Description
Toggle k-th bit of a number from Right in C++

### Checklist

- [yes ] I've read the contribution guidelines.
- [ yes] I've checked the issue list before deciding what to submit.
- [ yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2890

### CODE REVIEW

**Comment by mansi-9** [comment]
@bhav09 kindly review

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
 - [Sub-array with given sum](cp/SubArrayWithGivenSum.cpp)
 - [Suduko Solver](cp/SudukoSolver.cpp)
 - [Target Sum triplets](cp/target_sum_triplets.cpp)
+- [Toggle Kth Bit From Right](ToggleKthBitFromRight.cpp)
```
Comment (iamrajiv) [inline C-Plus-Plus/README.md]: @mansi-9 The path should be `- [Toggle Kth Bit From Right](cp/ToggleKthBitFromRight.cpp)`.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@mansi-9 Add a description of the problem and link to the problem.

File: C-Plus-Plus/README.md (hunk 1 above)
Comment (mansi-9) [reply C-Plus-Plus/README.md]: Sure. Thanx!!!

**Comment by mansi-9** [comment]
@iamrajiv did the changes sir.

**Review by iamrajiv — APPROVED** [review]

**Comment by iamrajiv** [comment]
@all-contributors please add @mansi-9 for her code

**Pull request merged**